tutorial/index.md

```diff
@@ -206,9 +206,52 @@ See [`pgroonga_score` function](../reference/functions/pgroonga-score.html) for
 
 ### Highlight {#highlight}
 
-TODO
+You can use `pgroonga_highlight_html` function to get your search keywords highlighted easily.
```

Comment:
```suggestion
You can use `pgroonga_highlight_html` function to get your specified search keywords highlighted easily.
```
It would be more explaining to add "specified" since you need to specify which search keywords to be highlighted.